modules/nodes-pods-autoscaling-custom-prometheus-config.adoc

```diff
@@ -0,0 +1,175 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-autoscaling-custom-prometheus-config_{context}"]
+= Configuring the custom metrics autoscaler to use {product-title} monitoring
+
+You can use the installed {product-title} monitoring as a source for the metrics used by the custom metrics autoscaler. However, there are some additional configurations you must perform.
+
+[NOTE]
+====
+These steps are not required for an external Prometheus source.
+====
+
+You must perform the following tasks, as described in this section:
+
+* Create a service account to get a token.
+* Create a role.
+* Add that role to the service account.
+* Reference the token in the trigger authentication object used by Prometheus.
+
+.Prerequisites
+
+* {product-title} monitoring must be installed.
```

Comment:
Another prerequisite is, that monitoring of user-defined workloads must be enabled in {product-title} monitoring: and paste a link here: http://file.rdu.redhat.com/mburke/OSDOCS-3048-custom-autoscaling-keda/monitoring/configuring-the-monitoring-stack.html#creating-user-defined-workload-monitoring-configmap_configuring-the-monitoring-stack